iOS/SkyService.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>
```

Comment:
This file is added probably because you have the old `.xcworkspace` from the old project when CocoaPods was being used.

Would you remove your local `.xcworkspace` by `rm iOS/SkyService.xcworkspace/` and add the commit to this PR?

<img width="1510" alt="image" src="https://github.com/getditto/demoapp-skyservice/assets/26117257/3ef45cf8-a18f-42cc-b4c1-f0014c43f4c0">